### PR TITLE
Added a home page 

### PR DESCRIPTION
**Pull request by Hugovs2000**
Notes:
- Fixed auth bugs
- Implemented the basic functionality of the home page (still more to come)
- Ignore the console logs for now.
- Api call only fetches a local json file for now
- Will implement state management next.
Screenshot 2024-06-20 at 13 03 03

### CODE REVIEW

**Review by Julianna-Venter — APPROVED** [review]
Looks good

File: src/app/components/home/home.component.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            [totalCost]="trip.totalCost"
+            (click)="handleTripClick(trip.title)" />
+        </div>
+      </div>
+    }
+    <button
+      class="btn btn-circle fixed bottom-4 right-4 border-2 border-blue-500">
+      +
+    </button>
+  </div>
```
Comment (yanevdw) [inline src/app/components/home/home.component.html:33]: You could potentially use ```w-3/5``` as opposed to an arbitrary value to set the width of the element to 60%.

**Review by yanevdw — APPROVED** [review]
Fantastic job, well done!

**Review by yanevdw — APPROVED** [review]

File: src/app/components/home/home.component.html (hunk 1 above)
Comment (Hugovs2000) [reply src/app/components/home/home.component.html:33]: Didn't notice that, thank you.

**Pull request merged**